conductr_cli/conduct_main.py

```diff
@@ -5,15 +5,22 @@
     conduct_stop, conduct_unload, conduct_version, conduct_logs,\
     conduct_events, conduct_acls, host, logging_setup
 from conductr_cli.constants import \
-    DEFAULT_PORT, DEFAULT_API_VERSION, DEFAULT_CLI_SETTINGS_DIR,\
+    DEFAULT_SCHEME, DEFAULT_PORT, DEFAULT_BASE_PATH, \
+    DEFAULT_API_VERSION, DEFAULT_DCOS_SERVICE, DEFAULT_CLI_SETTINGS_DIR,\
     DEFAULT_CUSTOM_SETTINGS_FILE, DEFAULT_CUSTOM_PLUGINS_DIR,\
     DEFAULT_BUNDLE_RESOLVE_CACHE_DIR, DEFAULT_WAIT_TIMEOUT
+from dcos import config
+from pathlib import Path
 from pyhocon import ConfigFactory
+from urllib.parse import urlparse
 import os
 import sys
 
 
-def add_ip_and_port(sub_parser):
+def add_scheme_ip_port_and_base_path(sub_parser):
+    sub_parser.add_argument('--scheme',
+                            help='The optional ConductR scheme, defaults to `http`',
```

Comment:
Shouldn't something as important as the orchestrator API default to https? (perhaps that's for the future?)